extensions/amp-story/1.0/test/test-amp-story.js

```diff
@@ -141,6 +149,21 @@ describes.realWin(
       expect(story.getPageCount()).to.equal(pagesCount);
     });
 
+    it('should return the correct number of pages with ads', async () => {
+      const pagesCount = 4;
+      const adId = 'i-amphtml-ad-page-1';
+      await createStoryWithPages(pagesCount, [
+        'cover',
+        'page-1',
+        'page-2',
+        'page-3',
+      ]);
+      await story.layoutCallback();
+      story.addPage(await createStoryAdPage(adId));
+      story.insertPage('page-2', adId);
+      expect(story.getPageCount()).to.equal(pagesCount);
```

Comment:
Re: this method is not used, we don't need to test it. Please remove this test (it only tests itself)